dEdX/src/dEdxAnalyser.cc

```diff
@@ -544,6 +670,7 @@ void dEdxAnalyser::processEvent( LCEvent * evt ) {
       {
         //int PFO_pdg = pfo->getParticleIDs()[0]->getPDG();
         int PFO_pdg = pidh->getParticleID(pfo,pidh->getAlgorithmID("dEdxPID")).getPDG();
+        //streamlog_out(MESSAGE) << PFO_pdg << std::endl;
         if (PFO_pdg==0) PFO_pdg=10;
         for (int j=0; j<_nPart+1; j++) if (pdg==_PDG[j])
           for (int k=0; k<_nPart+1; k++) if (PFO_pdg==_PDG[k])
```

Comment:
```suggestion
        for (int j=0; j<_nPart+1; j++)  {
          if (pdg !=_PDG[j]) continue;
          for (int k=0; k<_nPart+1; k++) {
            if (PFO_pdg !=_PDG[k]) continue;
            // ... whatever needs to be done here
          }
        }
```
This is not strictly part of the PR, but could this also be split into several separate lines using `{}` to scope things accordingly? Otherwise this is again very hard to read and even more error prone.